tests/web/rpc/v1alpha/test_trace_item_attribute_values.py

```diff
@@ -16,7 +16,7 @@
 from tests.helpers import write_raw_unprocessed_events
 
 BASE_TIME = datetime.utcnow().replace(minute=0, second=0, microsecond=0) - timedelta(
-    minutes=180
+    days=2, minutes=180
```

Comment:
I think this will just push the problem to the 2nd, where BASE_TIME will be the 30th again, and then base_time.days + 1 will be the 31st

Reply:
thanks!